tests/test_class_image.py

```diff
@@ -18,10 +18,12 @@ def setUp(self):
         '''Test a generic Image class'''
         self.image1 = Image('1234abcd')
         self.image2 = TestImage('5678efgh')
+        self.image3 = TestImage('5679efghi')
```

Comment:
Could you change the input sha for test `image3` to something that is 8 characters and slightly more distinguishable from `image2`? Maybe `wxyz0987` or something?

Reply:
Or maybe there is a reason you chose to do it this way which I am overlooking :)